core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -42,6 +42,8 @@ angular.module('oppia').directive('adminMiscTab', [
           '/explorationdataextractionhandler');
         var SEND_DUMMY_MAIL_HANDLER_URL = (
           '/sendDummyMailToAdminHandler');
+        var INTERACTIONS_BY_ID_HANDLER_URL = (
+          '/interactionsByExplorationId');
```

Comment:
Move this to the line above (i.e, you can fit 80 chars (max) in a line)

Reply:
updated